### PR TITLE
OSDOCS-6364: CSI-proxy support for Windows Containers

### PR DESCRIPTION
**Pull request by DCChadwick**
[OSDOCS-6364](https://issues.redhat.com//browse/OSDOCS-6364): CSI-Proxy Support for Windows Containers

Version(s): 4.14+

Issue: [OSDOCS-6364](https://issues.redhat.com//browse/OSDOCS-6364)

Link to docs preview:

https://69099--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/understanding-windows-container-workloads.html#windows-node-services_understanding-windows-container-workloads

QE review:
- [x] QE has approved this change.

Additional information:

The drivers are developed by a third-party, so we're just documenting a statement of support here.

### CODE REVIEW

**Comment by DCChadwick** [comment]
This PR replaces https://github.com/openshift/openshift-docs/pull/61573/files

**Comment by abrennan89** [comment]
@DCChadwick is this open against the right branch?
It's open against `main` but marked as 4.14 in the description. If this change is only for 4.14 rather than 4.14+, it should be opened against the `enterprise-4.14` branch instead of `main`.
Can you confirm please, thanks.

File: modules/windows-node-services.adoc [hunk 1]
```diff
@@ -37,4 +37,7 @@ The following Windows-specific services are installed on each Windows node:
 |containerd container runtime
 |Manages the complete container lifecycle.
 
+|CSI Proxy
+|Enables CSI drivers to perform storage operations on the node, which allows containerized CSI Drivers to run on Windows nodes.
```
Comment (abrennan89) [inline modules/windows-node-services.adoc:41]: Capitalize "drivers" consistently

**Review by abrennan89 — COMMENTED** [review]
Small comment about capitalization.
You also will likely need to go through the [change management](https://docs.google.com/document/d/1ofAtradc5MlQZaVFSeeQtYwYa6AGX3UJG6DaMyNLFm8/edit#heading=h.34s34dizqq4u) process, since this looks like it's removing something from published release notes?

File: modules/windows-node-services.adoc (hunk 1 above)
Comment (DCChadwick) [reply modules/windows-node-services.adoc]: Oops! Thanks for catching that!

**Comment by jeana-redhat** [comment]
LGTM, just need to know about [change management](https://github.com/openshift/openshift-docs/pull/69099#pullrequestreview-1771083356) for this one

**Comment by kalexand-rh** [comment]
LGTM

**Comment by arnaik-rh** [comment]
LGTM

**Comment by DCChadwick** [comment]
Merge review squad: This one is good to go. It has already passed QE, peer, and merge review, and I've now received all the change management ACKs and LGTMs I needed.

**Pull request merged**